src/app/shared/services/helper.service.ts

```diff
@@ -56,4 +56,11 @@ export class HelperService {
   createCompareTabId(originalReport: Report, runResultReport: Report): string {
     return `${originalReport.storageId}-${runResultReport.storageId}`;
   }
+
+  conditionalCssClass(item: CreateTreeItem): string {
```

Comment:
Create a util class and add this as a static method, please don't add anything to the helper service class, it is an abomination and will be removed eventually

Reply:
Added the change